docs/datafeed.md

```diff
@@ -0,0 +1,121 @@
+# Datafeed
+
+The datafeed service is a service used for handling the [_Real Time Events_](https://developers.symphony.com/restapi/docs/real-time-events). 
+For each interaction of a user within the IM, MIM or Room chat, an event will be sent to the datafeed.
+The bot can create a datafeed, list all created datafeeds or retrieve all the Real Time events within a datafeed through datafeed API.
+The datafeed service is a core service built on top of the Datafeed API and provide a dedicated contract to bot developers to work with datafeed. 
+
+The datafeed service is also the service where the [Activity API](../docs/activity-api.md) is built on top of.
```

Comment:
> ℹ️  For more advanced interactions between users and bots, you can also read [Activity API](../docs/activity-api.md)

Reply:
Do you want me to append this sentence to the one in my PR or completely replace it?